### ISSUE TITLE
Not displaying USB Camera image - No suitable resolutions found to fit 

### ISSUE THREAD

**Issue by maklimcz** [openDsh/dash#182]
## Hardware

| Device             | Connection Method | Screen                       | USB Device     | 
| ------------------ | ----------------- | ---------------------------- | ----------------- | 
| Raspberry Pi 4 4GB | USB Cable         | Chinese 5 inch screen | Chinese RCA-USB adapter |     

Installation Method: Install.sh 

## Description of problem:

USB Camera image not displaying on camera page when connect. Only white screen. Overlay disabled.
In logs 
```
No suitable resolutions found to fit in QSize(724, 480)
```
However my camera has 640x480 output. `gst-device-monitor-1.0` output:

```
Device found:

        name  : USB2.0 PC CAMERA
        class : Video/Source
        caps  : image/jpeg, width=640, height=480, pixel-aspect-ratio=1/1, framerate={ (fraction)60/1, (fraction)30/1, (fraction)15/1, (fraction)5/1 }
        properties:
                udev-probed = true
                device.bus_path = platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.1.4:1.0
                sysfs.path = /sys/devices/platform/scb/fd500000.pcie/pci0000:00/0000:00:00.0/0000:01:00.0/usb1/1-1/1-1.1/1-1.1.4/1-1.1.4:1.0/video4linux/video0
                device.bus = usb
                device.subsystem = video4linux
                device.vendor.id = 18ec
                device.vendor.name = ARKMICRO
                device.product.id = 5555
                device.product.name = USB2.0 PC CAMERA
                device.serial = ARKMICRO_USB2.0_PC_CAMERA
                device.capabilities = :capture:
                device.api = v4l2
                device.path = /dev/video0
                v4l2.device.driver = uvcvideo
                v4l2.device.card = USB2.0 PC CAMERA
                v4l2.device.bus_info = usb-0000:01:00.0-1.1.4
                v4l2.device.version = 394803 (0x00060633)
                v4l2.device.capabilities = 2225078273 (0x84a00001)
                v4l2.device.device_caps = 69206017 (0x04200001)
        gst-launch-1.0 v4l2src ! ...


```

## Expected Result:

Camera image displays correctly.

## Walkaround:
I've managed walkaround changing the pipeline in the source and recompiling camera.cpp.
https://github.com/openDsh/dash/compare/develop...maklimcz:dash:develop#diff-37c4b1554b048ac37e0a46f0275628bb7439221f8338f2f7cc2ea7f877b6ffab

**Comment by rsjudka** [comment]
yeah the camera interface has a lot to be desired 🙂 easiest would be to let you type your own in... but I kinda try to minimize keyboard input as much as possible (since not everyone has access to one in the car)

open to suggestions on how we can make this more customizable! (maybe like a "config" mode where more options are exposed? like when setting up your pi on a bench)

**Comment by robert5974** [comment]
A config mode does sound good.  Maybe combo boxes with the most common
resolutions but also an uploadable config file for custom settings.  I'll
think on it some more.

Robert Crowley

**Comment by maklimcz** [comment]
i think the best option is to specify whole pipeline command in config file
you configure that once and if it works there no reason to change it, unless you replace your camera

**Comment by monkeyboyohyeah** [comment]
Bumping this as without some sort of settings page the camera is not very useful to a lot of people. 
I'm guessing a lot of people are setting up their pi on a bench before installing into the car, so setting camera aspect ratio/screen size via command line or keyboard would be possible?
